.github/CODEOWNERS

```diff
@@ -52,3 +52,15 @@
 # Non-tutorial documentation text and infrastructure
 /doc @larsoner @drammock
 /logo @drammock
+
+# Current-source density
```

Comment:
I tried to keep `mne/` toward the top and misc/other stuff toward the bottom, could you move these up?